mne/surface.py

```diff
@@ -698,7 +708,39 @@ def read_surface(fname, read_metadata=False, return_dict=False, verbose=None):
     read_tri
     """
     fname = _check_fname(fname, 'read', True)
-    ret = _get_read_geometry()(fname, read_metadata=read_metadata)
+    _check_option('file_format', file_format, ['auto', 'freesurfer', 'obj'])
+
+    if file_format == 'auto':
+        _, ext = op.splitext(fname)
+        if ext.lower() == '.obj':
+            file_format = 'obj'
+        else:
+            file_format = 'freesurfer'
+
+    if file_format == 'freesurfer':
+        ret = _get_read_geometry()(fname, read_metadata=read_metadata)
+
+    elif file_format == 'obj':
+        coords = []
+        faces = []
+        with open(fname) as f:
+            for line in f:
+                line = line.strip()
+                if len(line) == 0 or line[0] == "#":
+                    continue
+                split = line.split()
+                if split[0] == "v":  # vertex
+                    coords.append([float(item) for item in split[1:]])
+                elif split[0] == "f":  # face
+                    dat = [int(item.split("/")[0]) for item in split[1:]]
+                    if len(dat) != 3:
+                        raise RuntimeError('Only triangle faces allowed.')
+                    # In .obj files, indexing starts at 1
+                    faces.append([d - 1 for d in dat])
+        ret = (np.array(coords), np.array(faces))
```

Comment:
can you put this in a private function to have the same design as with _get_read_geometry?